OpenSim/Tests/Components/testComponents.cpp

```diff
@@ -217,7 +217,8 @@ void testComponent(const Component& instanceToTest)
 
             if (dependency == nullptr){
                 // Get a concrete instance of a PhysicalFrame, which is a Body
-                if (dependencyTypeName == "PhysicalFrame"){
+                size_t n = dependencyTypeName.length();
+                if (n > 4 && dependencyTypeName.substr(n-5) == "Frame") {
```

Comment:
I agree, but here we do not even have an object, just the type name of the dependency.
Maybe there is a better way without using the registered types.